Cargo.toml

```diff
@@ -7,9 +7,12 @@ edition = "2018"
 [dependencies]
 anyhow = "1.0.40"
 cpal = "0.13.3"
+midir = "0.7.0"
```

Comment:
This is the cross-platform library that uses whatever backend is appropriate to get MIDI messages (as bytes).